plugins/modules/nd_setup.py

```diff
@@ -0,0 +1,471 @@
+        type: list
+        required: true
+        elements: dict
+        suboptions:
+          ntp_host:
+            description:
+            - The IP address of the NTP server.
+            type: str
+            required: true
+          ntp_key_id:
+            description:
+            - The NTP Key ID.
+            type: int
+          preferred:
+            description:
+            - The preferred status of the NTP host.
+            type: bool
+            default: false
+      keys:
+        description:
+        - This option is used for setting up the NTP keys.
+        type: list
+        elements: dict
+        suboptions:
+          ntp_key_id:
+            description:
+            - The NTP Key ID.
+            type: int
+            required: true
+          ntp_key:
+            description:
+            - The value of the NTP key.
+            type: str
+            required: true
+          authentication_type:
+            description:
+            - The authentication type for the NTP key.
+            type: str
+            choices: [ AES128CMAC, SHA1, MD5 ]
+            required: true
+          trusted:
+            description:
+            - The trusted status of the NTP key.
+            type: bool
+            default: false
+  nodes:
+    description:
+    - The node details to set up Nexus Dashboard and bring up the User Interface.
+    type: list
+    elements: dict
+    suboptions:
+      hostname:
+        description:
+        - The host name of the node.
+        type: str
+        required: true
+      serial_number:
+        description:
+        - The serial number of the node.
+        type: str
+        required: true
+      management_ip_address:
+        description:
+        - The management IP address of the node.
+        type: str
+        required: true
+      username:
+        description:
+        - The username of the node.
+        type: str
+        required: true
+      password:
+        description:
+        - The password of the node.
+        type: str
+        required: true
+      management_network:
+        description:
+        - The network that is used for DNS and NTP communication.
+        type: dict
+        required: true
+        suboptions:
+          ipv4_address:
+            description:
+            - The IPv4 address of the management network.
+            type: str
+            required: true
+          ipv4_gateway:
+            description:
+            - The IPv4 gateway of the management network.
+            type: str
+            required: true
+          ipv6_address:
+            description:
+            - The IPv6 address of the management network.
+            type: str
+          ipv6_gateway:
+            description:
+            - The IPv6 gateway of the management network.
+            type: str
+      data_network:
+        description:
+        - The network that is used for clustering and communication between sites and applications.
```

Comment:
The network that is used for clustering and communication between sites and applications. -> The network used for clustering and communication between sites and applications.

Reply:
ok